Marlin/src/HAL/HAL_STM32F1/SoftwareSerial.cpp

```diff
@@ -18,6 +18,12 @@
  */
 #if defined(__STM32F1__) && !defined(HAVE_SW_SERIAL)
 
+#include "../../inc/MarlinConfig.h"
+#if HAS_TMC220x
+  #warning "Consider using SoftwareSerialM with HAVE_SW_SERIAL and appropriate SS_TIMER defined.
```

Comment:
missing the ending "

Reply:
Thanks. I thought I had already pushed that fix while posting the PR. I’ll get it updated shortly.